example/index.html

```diff
@@ -18,11 +18,14 @@
     <script type="importmap">
       {
         "imports": {
-          "hello-csv": "https://cdn.jsdelivr.net/npm/hello-csv@0.1.1/dist/bundled/index.es.js"
+          "hello-csv": "https://cdn.jsdelivr.net/npm/hello-csv@0.1.2/dist/bundled/index.es.js"
         }
       }
     </script>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hello-csv@0.1.1/dist/bundled/hello-csv.css">
+    <link
```

Comment:
We should also change it in `InstallTabs.tsx::no build`

Reply:
https://github.com/HelloCSV/HelloCSV/pull/155/files#diff-f896d339f6e163b7e8de32c3a4b6f3ffcfb33511c6a5e6b78a537456fa01bd09R23